go.mod

```diff
@@ -23,14 +23,14 @@ require (
 	github.com/evanphx/json-patch v5.6.0+incompatible
 	github.com/getsentry/sentry-go v0.11.0
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
-	github.com/go-git/go-billy/v5 v5.4.1
```

Comment:
@daskuznetsova  Do not update all dependencies. It might tamper some other functionalities.

Reply:
I think there is no other way than updating these.